MDCB 2.9.0 RNs.mdx (#1442)
TykTechnologies/tyk-docs: developer-support/release-notes/mdcb.mdx

---

## 2.9 Release Notes

### 2.9.0 Release Notes

#### Release Date 6 March 2026

#### Release Highlights

> In this patch release, we've improved visibility into your data planes by including the specific APIs and Policies loaded by each gateway in the MDCB `/dataplanes` endpoint. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v2.9.0).

Reviewer: ```suggestion
In this release, we've added visibility of the lists of APIs and Policies loaded by each Data Plane Gateway connected to MDCB.

For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v2.9.0).
```

| :------------------------------- | :--------------- | :------------------- | :-------- |
| [Redis](https://redis.io/download/)    | 6.2.x, 7.x, 7.4.x      | 6.2.x, 7.x, 7.4.x      | |
| [Valkey](https://valkey.io/download/)  | 7.2.x, 8.0.x, 8.1.x    | 7.2.x, 8.0.x, 8.1.x    | |
| [MongoDB](https://www.mongodb.com/try/download/community)  | 5.0.x, 6.0.x, 7.0.x | 4.4.x, 5.0.x, 6.0.x, 7.0.x | |
| [PostgreSQL](https://www.postgresql.org/download/)         | 13.x - 17.x        | 13.x - 17.x            | |

Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.

#### Deprecations
There are no deprecations in this release.

#### Upgrade instructions
If you are upgrading to 2.9.0, please follow the detailed [upgrade instructions](#upgrading-tyk).

#### Downloads
- [Docker image v2.9.0](https://hub.docker.com/r/tykio/tyk-mdcb-docker/tags?page=&page_size=&ordering=&name=v2.9.0)
  - ```bash
    docker pull tykio/tyk-mdcb-docker:v2.9.0
    ```
#### Changelog
<a id="Changelog-v2.9.0"></a>

##### Added

<AccordionGroup>

> <Accordion title='Enhanced MDCB Dataplane Endpoint with Loaded API and Policy Information'>

Reviewer: ```suggestion
<Accordion title='Enhanced MDCB `/dataplane` Endpoint with Information on Loaded APIs and Policies'>
```

> <Accordion title='Support Enhanced MDCB `/dataplane` Endpoint with Information on Loaded APIs and Policies'>

Reviewer: ```suggestion
<Accordion title='Visibility of the APIs and Policies loaded by each Data Plane Gateway'>
```

> The MDCB `/dataplanes` endpoint has been enhanced to provide visibility into your Tyk deployment. The endpoint response now includes a detailed list of the specific APIs and Policies loaded on each connected data plane gateway. Additionally, you can now filter the results for a specific gateway by using the new `node_id` query parameter. This allows for easier monitoring and troubleshooting of your environment by providing a clear picture of what is running on each gateway.

Reviewer: ```suggestion
The MDCB `/dataplanes` endpoint has been enhanced to provide visibility into your Tyk deployment. This now returns a detailed list of the APIs and policies loaded on each connected data plane gateway. You can retrieve the results for a specific Gateway using the `node_id` query parameter. This update provides a clear picture of what is running on each Gateway, simplifying monitoring and troubleshooting of your deployed Data Planes.
```

> </Accordion>

Reviewer 2: ```suggestion
<Accordion title='Enhanced MDCB Dataplane Endpoint with Loaded API and Policy Information'>
The MDCB `/dataplanes` endpoint has been enhanced to provide visibility into your Tyk deployment. The endpoint response now includes a detailed list of the specific APIs and Policies loaded on each connected data plane gateway. Additionally, you can now filter the results for a specific gateway by using the new `node_id` query parameter. This allows for easier monitoring and troubleshooting of your environment by providing a clear picture of what is running on each gateway.
</Accordion>
```

> Gateway is updated to support the enhanced MDCB `/dataplanes` endpoint. The enhanced MDCB endpoint will provide visibility into your Tyk deployment. This now returns a detailed list of the APIs and policies loaded on each connected data plane gateway. You can retrieve the results for a specific Gateway using the `node_id` query parameter. This update provides a clear picture of what is running on each Gateway, simplifying the monitoring and troubleshooting of your deployed Data Planes.

Reviewer: ```suggestion
The `GET /dataplanes` endpoint has been extended to provide enhanced visibility of your Tyk deployment. It now returns a list of the APIs and policies loaded on each connected Data Plane Gateway. You can retrieve the results for a specific Gateway using the `node_id` query parameter.

This update provides a clear picture of what is running on each Gateway, simplifying the monitoring and troubleshooting of your Tyk deployment.
```
